eras/conway/impl/src/Cardano/Ledger/Conway/Governance/DRepPulser.hs

```diff
@@ -298,20 +316,31 @@ class
   runConwayRatify globals ratifyEnv ratifyState (RatifySignal ratifySig) =
     let ratifyResult =
           runReader
-            ( applySTS @(ConwayRATIFY era) (TRC (ratifyEnv, ratifyState, RatifySignal $ reorderActions ratifySig))
+            ( applySTS @(ConwayRATIFY era) $
+                TRC (ratifyEnv, ratifyState, RatifySignal $ reorderActions ratifySig)
             )
             globals
      in case ratifyResult of
           Left ps ->
```

Comment:
```suggestion
          Left (x :| _) -> absurd x
```

Reply:
I did not fully understand the intention and motivation @Soupstraw, sorry. Could you explain a bit?

Reply:
`absurd` ensures at compile time that it is an impossible case, so you don't need to provide any error messages. This is possible because:
```haskell
type PredicateFailure (ConwayRATIFY era) = Void
```